### DISCUSSION TITLE
A/B Testing Emails

### DISCUSSION THREAD

**Discussion by psugihara** [sofn-xyz/mailing#105] [Polls]
Hey y'all. We're stoked that the community is growing and people are finding the tool useful. Turns out a lot of people want email dev to be more fun 🌞.

Transparently we're thinking about how one might build a mailing adjacent business to allow us to continue working on mailing sustainably. We're all in on open-source, and also capitalists with children to feed.

So the open ended questions... **Do you A/B test email content? How do you know if your emails are _working_?**

Also, feel free to shoot me an email about this if you'd rather [peter@campsh.com](mailto:peter@campsh.com)

Tagging a few of you who have been godsends in the issues: @mrlubos, @neuxregime, @sebasalvarado, @alan-francis, @jdotjdot

Thanks in advance,
Mailing Core Team (@monicakogler, @alexfarrill, @psugihara)

**Comment by mrlubos** [comment]
We don't do A/B testing and haven't really considered it yet. It would add unnecessary complexity, we don't have dedicated resources to focus on it, and it's not a core part of our business. We do care about the open rates, however. For that, we use SendGrid data (our provider) and look at open rates.

Sadly, SendGrid doesn't keep detailed data for more than 30 days, so there's some complexity added by having to store this information ourselves. This part really sucks and it shouldn't be taking up such a huge chunk of our codebase.

Sorry if my answer isn't clear. I think we would love to A/B test as we care about open rates, but we don't have a specialist on our team and no one explored how much setup it would be to create a productive workflow. As a result, we update our emails very rarely. Most of them are of transactional nature and we want people to open those. For example, when we post an update, we send an email saying "hey, here is what's new."
